<?xml version="1.0" encoding="UTF-8"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1478097449231" stop="1478097459237">
	<name>a</name>
	<title>a</title>
	<test-cases>
		<test-case start="1478097449232" stop="1478097459236" status="broken">
			<name>sasd</name>
			<title>sasd</title>
			<labels>
			</labels>
			<parameters>
				<parameter kind="environment-variable" name="capabilities" value="{&quot;browserName&quot;:&quot;phantomjs&quot;}"/>
				<parameter kind="environment-variable" name="spec files" value="[&quot;/Users/christianbromann/Sites/Webprojekte/webdriverjs/DEV/a.js&quot;]"/>
			</parameters>
			<steps>
				<step start="1478097449240" stop="1478097449716" status="passed">
					<name>POST /wd/hub/session/04018059-22b6-4d06-b096-b5d3490cfdbf/url</name>
					<title>POST /wd/hub/session/04018059-22b6-4d06-b096-b5d3490cfdbf/url</title>
					<attachments>
						<attachment title="Request" source="6c974411-a4c0-45bf-9f8e-b911a289f7e5-attachment.json" type="application/json" size="57"/>
						<attachment title="Response" source="f4c669c0-ac11-4304-b30d-1aaac6338825-attachment.json" type="application/json" size="196"/>
					</attachments>
					<steps>
					</steps>
				</step>
				<step start="1478097449760" stop="1478097449808" status="passed">
					<name>POST /wd/hub/session/04018059-22b6-4d06-b096-b5d3490cfdbf/elements</name>
					<title>POST /wd/hub/session/04018059-22b6-4d06-b096-b5d3490cfdbf/elements</title>
					<attachments>
						<attachment title="Request" source="229d37bb-78a8-433e-9b3f-4dbf0a9aa6f8-attachment.json" type="application/json" size="51"/>
						<attachment title="Response" source="65c14449-c075-4478-acf2-d5c03cfa2b0e-attachment.json" type="application/json" size="1657"/>
					</attachments>
					<steps>
					</steps>
				</step>
			</steps>
			<attachments>
			</attachments>
			<failure>
				<message>timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.</message>
				<stack-trace>Error: timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.
    at Timeout.&lt;anonymous&gt; (/Users/christianbromann/Sites/Webprojekte/webdriverjs/wdio-mocha-framework/node_modules/mocha/lib/runnable.js:232:19)
    at tryOnTimeout (timers.js:224:11)
    at Timer.listOnTimeout (timers.js:198:5)</stack-trace>
			</failure>
		</test-case>
	</test-cases>
</ns2:test-suite>